### PR TITLE
ARM compute images on batch

### PR DESCRIPTION
**Pull request by tilne**
* Make dependencies installed in Batch Dockerfile architecture agnostic
* Modify image build accordingly when using ARM compute instances

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/tox.ini [hunk 1]
```diff
@@ -211,7 +211,7 @@ changedir =
     ../cloudformation
 deps = cfn-lint
 commands =
-    cfn-lint --info *.cfn.json --ignore-templates fsx-substack.cfn.json
+    cfn-lint --info *.cfn.json --ignore-templates fsx-substack.cfn.json --ignore-checks E3030
```
Comment (demartinofra) [inline cli/tox.ini:214]: if `E3030` cannot be fixed can we add a comment explaining why?

Comment (tilne) [reply cli/tox.ini:217]: Done. Linked to [the issue for it](https://github.com/aws-cloudformation/cfn-python-lint/issues/1588).

**Comment by tilne** [comment]
Clicking through to the travis status link shows that all tests have passed. Because of this and the fact that this is being merged into a feature branch and not develop I'm going to go ahead and merge.

File: cli/tox.ini (hunk 1 above)
Comment (PatMyron) [reply cli/tox.ini:217]: Thanks for the report! [Rules can also be ignored at a resource level](https://github.com/aws-cloudformation/cfn-python-lint/#resource-based-metadata) if you just want to ignore it for one resource

Comment (tilne) [reply cli/tox.ini:217]: I wasn't aware of that. Thanks for the tip.

**Pull request merged**